app/views/task_lists/edit.html.erb

```diff
@@ -2,6 +2,12 @@
 
 <%= link_to("<< Back to List", task_lists_path, :class => 'back-link') %>
```

Comment:
I'm getting an error when trying to go back to the list: `Couldn't find TaskList with 'id'=4` (4 is the task's id, not the list's id).

Reply:
I think this is fixed